tests/Rector/Class_/RenameClassRector/RenameClassRectorTest.php

```diff
@@ -17,17 +18,29 @@
  */
 final class RenameClassRectorTest extends AbstractRectorTestCase
 {
-    public function test(): void
+    /**
+     * @dataProvider provideTestFiles
+     */
+    public function test(string $filePath): void
+    {
+        $this->doTestFiles([$filePath]);
+    }
+
+    public function provideTestFiles(): Iterator
     {
-        $this->doTestFiles([
-            __DIR__ . '/Fixture/class_to_new.php.inc',
-            __DIR__ . '/Fixture/class_to_interface.php.inc',
-            __DIR__ . '/Fixture/interface_to_class.php.inc',
-            __DIR__ . '/Fixture/name_insensitive.php.inc',
-            __DIR__ . '/Fixture/twig_case.php.inc',
-            __DIR__ . '/Fixture/underscore_doc.php.inc',
-            __DIR__ . '/Fixture/keep_return_tag.php.inc',
-        ]);
+        yield [__DIR__ . '/Fixture/class_to_new.php.inc'];
+        yield [__DIR__ . '/Fixture/class_to_interface.php.inc'];
+        yield [__DIR__ . '/Fixture/interface_to_class.php.inc'];
+        yield [__DIR__ . '/Fixture/name_insensitive.php.inc'];
+        yield [__DIR__ . '/Fixture/twig_case.php.inc'];
+        yield [__DIR__ . '/Fixture/underscore_doc.php.inc'];
+        yield [__DIR__ . '/Fixture/keep_return_tag.php.inc'];
+        yield [__DIR__ . '/Fixture/rename_class_without_namespace.php.inc'];
```

Comment:
There should comment that describe set bellow, so it's easier to debug later:

```php
// rename class with namespace
yield [__DIR__ . '/Fixture/rename_class_without_namespace.php.inc'];
...
```

Reply:
What do you mean? Debug what? You will see exactly which data set failed and file with fixture contains exactly what it does `rename_class_without_namespace.php.inc`.

I do not understand what added value would this comment add.

Reply:
There are always 2 groups of test fixtures that should be clearly separated.

Those that change file and those that should ignore it.